### PR TITLE
Fix/nomenclature updates

### PR DESCRIPTION
**Pull request by jeffersonrabb**
### All Submissions:

* [x] Have you followed the [Newspack Contributing guideline](https://github.com/Automattic/newspack-plugin/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) and [VIP Go coding standards](https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Applies recent naming changes to match the Newspack Plugin Campaigns Wizard. "Campaigns" are renamed to "Prompts." "Campaign Groups" are renamed "Campaigns." All of the changes in this branch apply to UI text and comments only. No function names or variables have been changed.

### How to test the changes in this Pull Request:

Go through the UI, see if you can find any incorrect references to "Campigns" or "Campaign Groups."

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?

### CODE REVIEW

File: src/editor/SegmentationSidebar.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ const SegmentationSidebar = ( { onMetaFieldChange, selected_segment_id } ) => {
 				href="/wp-admin/admin.php?page=newspack-popups-wizard#/segmentation"
```
Comment (dkoo) [inline src/editor/SegmentationSidebar.js:42]: Need to update this link to `#/segments` now that we've changed the route in https://github.com/Automattic/newspack-plugin/pull/833.

**Review by dkoo — CHANGES_REQUESTED** [review]
Just found one un-updated link for the renamed "Segments" tab. Other than that, all mentions of "Campaigns" and "Campaign Groups"/"Groups" in the UI seem to be correctly updated to "Prompts" and "Campaigns".

It'll be another and bigger task to update all of our variable and function names!

File: src/editor/SegmentationSidebar.js (hunk 1 above)
Comment (jeffersonrabb) [reply src/editor/SegmentationSidebar.js]: Nice spot! Fixed in https://github.com/Automattic/newspack-popups/pull/431/commits/a48c8d91379e39f05719be3f2ae4f115b6aebe62

**Review by dkoo — APPROVED** [review]
Looks good now! Let me know if you need a re-review after fixing the conflicts.

**Comment by jeffersonrabb** [comment]
> Let me know if you need a re-review after fixing the conflicts.

If you wouldn't mind, I think it would be wise for you to give another look. The merge conflicts were fairly substantial and mostly due to `feat/segment-prioritization`, and I want to be sure I didn't break anything.

**Review by dkoo — APPROVED** [review]
@jeffersonrabb I found one more instance where I thought we could update the nomenclature, pushed as f8c7a2f. If that looks good to you, this is 👍

**Comment by jeffersonrabb** [comment]
> I found one more instance where I thought we could update the nomenclature

I noted that one and figured we could go either way since it could be interpreted to mean "disable (the) Campaigns (plugin) on this page." But your fix is good, let's keep it.

**Pull request merged**